rose/server/player.py

```diff
@@ -4,6 +4,25 @@
 class Player(object):
 
     def __init__(self, name, car, lane):
+        """
+            Creates a new Driver object.
+            Args:
+                name (str): The unique name of the driver for display.
+                car (int): A number from 0 to 3, representing the car type.
+                lane (int): The initial lane number, between 0 and the
+                             maximum lane.
+
+            Attributes:
+                x (int, optional): The X-coordinate of the driver.
+                                    Starts in the middle of the lane.
+                y (int, optional): The Y-coordinate of the driver.
+                                    Starts at 2/3 of the screen height.
+                action (str, optional): The current driver action. Defaults
+                                         to 'none'.
+                response_time (float, optional): The duration the driver takes
+                                                  to react. Starts as None.
+                score (int, optional): The driver's current score. Begins at 0.
+        """
```

Comment:
Indentation is off again - the entire contents is indented too much.

It should be:

```python
    def __init__(self, name, car, lane):
        """
        Creates a new Driver object.
        Args:
            name (str): The unique name of the driver for display.
            ...

        Attributes:
            x (int, optional): The X-coordinate of the driver.
                Starts in the middle of the lane.
            ...
        """
```

See https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats for all 4(!) docstring formats.

Reply:
fixed, 3rd time ice cream :ice_cream:

Reply:
in flake8 we trust ... need some pydoc checks ...